python/saltedge.py

```diff
@@ -44,6 +44,22 @@ def generate_headers(self, expire):
             'Secret': self.secret
         }
 
+    def verify(self, message, signature):
+        """
+        Verifies the signature on a message.
+        :param message: string, The message to verify.
+        :param signature: string, The signature on the message.
+        :return:
+        """
+        x509 = crypto.X509()
+        x509.set_pubkey(self._private_key)
```

Comment:
@mtkachenk0 Seems weird to set_pubkey as self._private_key

Reply:
@mtkachenk0 Using this same method but instead and `x509.set_pubkey(self._spectre_pub_key)` where the public key was loaded as 
```        
with open(spectre_pub_path, "rb") as pub_key:
        pub_key_data = pub_key.read()
        self._spectre_pub_key = crypto.load_publickey(crypto.FILETYPE_PEM, pub_key_data)
```

But that gave me the following error: `OpenSSL.crypto.Error: [('rsa routines', 'RSA_padding_check_PKCS1_type_1', 'invalid padding'), ('rsa routines', 'rsa_ossl_public_decrypt', 'padding check failed')]` Can you verify that you don't get this error when you run tests?

Reply:
@kueben sure